### PR TITLE
n-1 Satellite and Capsule upgrade support

### PR DESCRIPTION
**Pull request by jyejare**
The PR includes:

'n-1' capsule upgrade with satellite at 'n' version. 

Basically its like 'satellite' at latest downstream(zStream) and 'capsule' at released zStream.

### CODE REVIEW

**Comment by jyejare** [comment]
After reviewing this PR, please review https://github.com/SatelliteQE/robottelo-ci/pull/571, which is dependent on this PR.

File: automation_tools/satellite6/upgrade/capsule.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 logger = logger()
 
 
-def satellite6_capsule_setup(sat_host, os_version):
+def satellite6_capsule_setup(sat_host, os_version, n_minus1=False):
```
Comment (sghai) [inline automation_tools/satellite6/upgrade/capsule.py]: can we update `n_minus1` to `capsule_update` or `update_capsule`?

**Comment by sghai** [comment]
ACK pending a minor change.

**Comment by jyejare** [comment]
@sghai Worked on your comment, and changed the parameter behavior ```i.e by default to True now``` as I changed parameter from n-1 cpasule to upgradable_capsule.

**Comment by sghai** [comment]
ACK

**Pull request merged**